tests/js/server/dump/dump-cluster.js

```diff
@@ -280,6 +280,70 @@ function dumpTestSuite () {
         assertEqual({ value: [ i, i ] }, doc.more);
       }
     },
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test padded keygen
+////////////////////////////////////////////////////////////////////////////////
+
+    testKeygenPadded : function () {
+      var c = db._collection("UnitTestsDumpKeygenPadded");
+      var p = c.properties();
+
+      assertEqual(2, c.type()); // document
+      assertFalse(p.waitForSync);
+      assertEqual("padded", p.keyOptions.type);
+      assertFalse(p.keyOptions.allowUserKeys);
+      assertEqual(process.env['paddedLastValue'], p.keyOptions.lastValue);
+
+      assertEqual(1, c.getIndexes().length); // just primary index
+      assertEqual("primary", c.getIndexes()[0].type);
+      assertEqual(1001, c.count());
+
+      let allDocs = {};
+      c.toArray().forEach(doc => {
+        allDocs[doc.value] = doc;
+      });
+      let lastKey = "";
+      for (var i = 0; i < 1000; ++i) {
+        var doc = allDocs[i];
+
+        assertTrue(doc._key > lastKey, doc._key + ">" + lastKey);
+        assertEqual(i, doc.value);
+        assertEqual({ value: [ i, i ] }, doc.more);
+        lastKey = doc._key;
+      }
+      doc = c.save({});
```

Comment:
```suggestion
      let doc = c.save({});
```

Reply:
Or change the naming here, because it has the same naming as inside the for loop.